task-1.2/index.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const csv = require('csvtojson');
+const { pipeline } = require('stream');
+
+const csvFilePath = './task-1.2/csv/task-1.2.csv';
+const outputFilePath = './task-1.2/csv/task-1.2.txt';
+
+
+pipeline(
+	fs.createReadStream(csvFilePath),
+	csv(),
+	fs.createWriteStream(outputFilePath),
+	(err) => {
```

Comment:
Consider extracting pipeline's callback into a separate function. Such callbacks could be (and usually are) reused between different streams.

Reply:
That's done